interpreter/namerd/src/main/scala/io/buoyant/namerd/iface/NamerdHttpInterpreterInitializer.scala

```diff
@@ -27,7 +28,7 @@ case class NamerdHttpInterpreterConfig(
   dst: Option[Path],
   namespace: Option[String],
   retry: Option[Retry],
-  tls: Option[ClientTlsConfig]
+  tls: Option[TlsClientConfig]
```

Comment:
Just to confirm my understanding: this is a breaking change of something that was already broken, right?  Namerd's http interface never supported TLS on the server side?